oneapp-server/types/ApplicationProgramInfo.js

```diff
@@ -13,25 +13,25 @@ const typeDef = gql`
     "Applying for General Assistance"
     IS_GA_SELECTED: Boolean
     "Has active SNAP case"
-    HAVE_ACTIVE_CASE_CURRENTLY: Boolean
+    HAVE_ACTIVE_CASE_CURRENTLY: Boolean!
     "Case number(s) of active SNAP applications"
     CURRENT_CASE_NUMBERS: String
     "Has received SNAP previously"
-    HAD_ACTIVE_CASE_PREVIOULSY: Boolean
+    HAD_ACTIVE_CASE_PREVIOULSY: Boolean!
     "Case number(s) of previous SNAP applications"
     PREVIOUS_CASE_NUMBERS: String
```

Comment:
There are additional validation to capture all throughout here. For example, PREVIOUS_CASE_NUMBERS has a regex validation: https://github.com/newjersey/dhs-oneapp-legacy/blob/master/NJOneAppAuto/Forms/Application/Programs.aspx#L483